pyriemann/sampling.py

```diff
@@ -0,0 +1,317 @@
+import numpy as np
```

Comment:
Can you add tests for this new module?
But, they should be written after merge of #136 @sylvchev , and rebase of your fork.

Reply:
Yes, sure. I will wait for it then.

Reply:
Tests refactoring has been merged.